crates/meilisearch/src/routes/indexes/search_test.rs

```diff
@@ -0,0 +1,26 @@
+#[cfg(test)]
+pub mod search_test {
```

Comment:
Here it would be better to remove this block and instead put the cfg(test) while importing the file